specification/search/data-plane/Microsoft.Azure.Search.Service/stable/2019-05-06/examples/SearchServiceGetIndexerStatus.json

```diff
@@ -43,17 +43,38 @@
               {
                 "key": "",
                 "errorMessage": "Document key cannot be missing or empty.",
-                "statusCode": 400
+                "statusCode": 400,
+                "name": null,
+                "details": null,
+                "documentationLink": null
+              },
+              {
+                "key": "document id 1",
+                "errorMessage": "Could not read the value of column 'foo' at index '0'.",
+                "statusCode": 400,
+                "name": "DocumentExtraction.AzureBlob.MyDataSource",
+                "details": "The file could not be found.",
+                "documentationLink": "https://contoso.example.org?type=error"
               }
             ],
             "warnings": [
               {
                 "key": "document id",
-                "message": "A warning doesn't stop indexing, and is intended to inform you of certain interesting situations, like when a blob indexer truncates the amount of text extracted from a blob."
+                "message": "A warning doesn't stop indexing, and is intended to inform you of certain interesting situations, like when a blob indexer truncates the amount of text extracted from a blob.",
+                "name": null,
+                "details": null,
+                "documentationLink": null
+              },
+              {
+                "key": "document id 2",
+                "message": "Another interesting scenario happened.",
+                "name": "Enrichment.LanguageDetectionSkill.#4",
+                "details": "The skill did something that didn't break anything, nonetheless something we didn't expect happened, so it might be worth double checking.",
+                "documentationLink": "https://contoso.example.org?type=warning"
```

Comment:
From this example, it looks as if the customer somehow provides the `documentationLink` via the custom Web API skill. Is that the case, or are these links supposed to be generated internally by us? If it's the latter, the examples should not use the contoso host/domain.

Reply:
The latter, I'll use an actual link that we'll send (it's still going to be a forward link)